docs/source/credentials.rst

```diff
@@ -0,0 +1,39 @@
+Credentials
+***********
+
+Integration with Git
+====================
+
+Git and DataLad can use each other's credential system.
```

Comment:
It's probably good to mention up from whether either method can be used, or if /both/ are needed and work in tandem.

After reading all of this file, I realized I was unsure about whether both configurations are needed, or if it's one vs the other. I came back to the beginning hoping for a clarification, but didn't find it.

By now, I'm pretty convinced that both are needed, but it feels circular (git says talk to DataLad and DataLad says talk to git). The opening text should make the situation clear.

Reply:
Depends on what you mean by "needed". Both directions work independently and can stand on their own. What's needed depends on what you want. If you want to seemlessly use the credentials provided by both systems, no matter whether that's a `git clone` or a `datalad get`, then yes - you likely want both. But probably not both for the same URL.

In case of our datastores, you areadly provide credentials via a git-credential helper. So, all that's needed to make it complete, is telling DataLad to ask Git (when downloading content) when it encounters such URLs.

Will try to make that more clear.

Reply:
Ok. I think I see how this works now. The intro could benefit by making one thing explicit: these methods are useful if either DataLad or Git are already configured to discover credentials to access a remote resource. Then the following steps explain how to direct one to the other to get that info.

Obviously that is not exactly the smoothest way of saying it.

I recommend having the intro explain the overall concept. Then one section for each method.